### ISSUE TITLE
Just a question

### ISSUE THREAD

**Issue by kaelgabriel** [pawan47/Fetchreach_gym#3]
Have you got any success with training with PPO?

**Comment by pawan47** [comment]
No. I tried with PPO, but it was not learning anything in case of the simple inverted pendulum.
